src_erl/Communication_Layer/http_Nerlserver/src/source/sourceStatem.erl

```diff
@@ -78,25 +80,27 @@ state_name(_EventType, _EventContent, State = #source_statem_state{}) ->
 
 
 %%This cast receive a list of samples to load to the records csvList
-idle(cast, {csvList,Workers,CSVPath}, State = #source_statem_state{chunkSize = ChunkSize, myName = Myname, msgCounter = Counter, portMap = PortMap}) ->
+idle(cast, {csvList,Workers,CSVPath}, State = #source_statem_state{chunkSize = ChunkSize, myName = MyName, msgCounter = Counter, nerlnetGraph = NerlnetGraph}) ->
```

Comment:
lol
myName? whose name? 
Please be more specific what name it is since it is very confusing 
(at least documentation if you don't want to change the argument's name)